.github/workflows/ci.yml

```diff
@@ -0,0 +1,116 @@
+name: ci
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  test-linux:
+    runs-on: ubuntu-24.04
```

Comment:
I'm wondering if we should just use `ubuntu-latest`.  We often get people finding issues with updated distros with newer LLVM/clang versions, and it'd be nicer to catch that in CI rather than having users need to open an issue, even if CI might suddenly break.

Reply:
@kkysen Sure, I can do that.